tests/Application/templates/bundles/SyliusShopBundle/Contact/request.html.twig

```diff
@@ -15,4 +15,11 @@
 
 {% block javascripts %}
     <script src="https://www.google.com/recaptcha/api.js" async defer></script>
+
```

Comment:
Create twig file and include it, add to readme

Reply:
DONE